### ISSUE TITLE
Adding support fort left/right joystick push/press ?

### ISSUE THREAD

**Issue by mixaal** [mart1nro/joycontrol#87]
Hi, 

I love this project since and I use it as a joycon replacement. The last thing I'd love to see is the left/right joystick press. It's used in some games for quite important things like "Lock on the powerful foe" in Hyrule Warriors .

**Comment by choss** [comment]
How about the commands: 'r_stick' and 'l_stick' ?

**Comment by mixaal** [comment]
Yes, I know they are there and use them to put the X,Y value, but can they do the press ?

**Comment by mixaal** [comment]
Oh, maybe `queue.add('press r_stick')` ?

**Comment by indykoning** [comment]
having made https://github.com/indykoning/joycon-controller
i can verify it has support for it, as you csan see in: https://github.com/mart1nro/joycontrol/blob/18a09da1a04306534ff9e1df8a1a69c0192a3244/joycontrol/controller_state.py#L81

The command would be press/push l_stick
